CHANGELOG_UNRELEASED.md

```diff
@@ -0,0 +1,5 @@
+## Unreleased
+
+### Added
+
+- Exposed the underlying `WebView#displayWithHybridComposition` parameter.
```

Comment:
Eu tentaría explicar que aporta ese parámetro ou a razón pola que se expón

Reply:
Si, no changelog público pondrei máis chicha.